sdk/monitor/Microsoft.OpenTelemetry.Exporter.AzureMonitor/src/AzureMonitorTraceExporterEventSource.cs

```diff
@@ -4,14 +4,17 @@
 using System;
 using System.Collections.Generic;
 using System.Diagnostics.Tracing;
+using Azure.Core.Shared;
 
 namespace Microsoft.OpenTelemetry.Exporter.AzureMonitor
 {
     [EventSource(Name = EventSourceName)]
     internal sealed class AzureMonitorTraceExporterEventSource : EventSource
     {
-        private const string EventSourceName = "OpenTelemetry-TraceExporter-AzureMonitor";
+        private const string EventSourceName = "Microsoft-OpenTelemetry-TraceExporter-AzureMonitor";
```

Comment:
is this for only the TraceExporter or intended for all exporters?

Reply:
This is intended to capture debug logs from all types of exporters which is going to be a part of this project.

Reply:
should the name be changed to reflect the intent? 
"TraceExporter" -> "Exporter"

Reply:
Renamed EventSource.